modules/yieldoneBidAdapter.js

```diff
@@ -167,6 +171,107 @@ export const spec = {
   },
 }
 
+/**
+ * NOTE: server side does not yet support multiple formats.
+ * @param  {Object} bidRequest -
+ * @param  {boolean} [enabledOldFormat = true] - default: `true`.
+ * @return {string|null} - `"banner"` or `"video"` or `null`.
+ */
+function getMediaType(bidRequest, enabledOldFormat = true) {
+  let hasBannerType = Boolean(deepAccess(bidRequest, 'mediaTypes.banner'));
+  let hasVideoType = Boolean(deepAccess(bidRequest, 'mediaTypes.video'));
+
+  if (enabledOldFormat) {
+    hasBannerType = hasBannerType || bidRequest.mediaType === BANNER ||
+      (isEmpty(bidRequest.mediaTypes) && isEmpty(bidRequest.mediaType));
+    hasVideoType = hasVideoType || bidRequest.mediaType === VIDEO;
+  }
+
+  if (hasBannerType && hasVideoType) {
+    const playerParams = deepAccess(bidRequest, 'params.playerParams')
+    if (playerParams) {
+      return VIDEO;
+    } else {
+      return BANNER;
+    }
+  } else if (hasBannerType) {
+    return BANNER;
+  } else if (hasVideoType) {
+    return VIDEO;
+  }
+
+  return null;
+}
+
+/**
+ * NOTE:
+ *   If `mediaTypes.banner` exists, then `mediaTypes.banner.sizes` must also exist.
+ *   The reason for this is that Prebid.js will perform the verification and
+ *   if `mediaTypes.banner.sizes` is inappropriate, it will delete the entire `mediaTypes.banner`.
+ * @param  {Object} bidRequest -
+ * @param  {Object} bidRequest.banner -
+ * @param  {Array<string>} bidRequest.banner.sizes -
+ * @param  {boolean} [enabledOldFormat = true] - default: `true`.
+ * @return {string} - strings like `"300x250"` or `"300x250,728x90"`.
+ */
+function getBannerSizes(bidRequest, enabledOldFormat = true) {
+  let sizes = deepAccess(bidRequest, 'mediaTypes.banner.sizes');
+
+  if (enabledOldFormat) {
+    sizes = sizes || bidRequest.sizes;
+  }
+
+  const result = parseSizesInput(sizes).join(',');
+  return result;
```

Comment:
Following the other comments, might be cleaner to simply `return parseSizesInput(sizes).join(',');`